dallinger/experiment.py

```diff
@@ -130,11 +132,14 @@ def __init__(self, session=None):
             self.configure()
 
         try:
+            location = type(self).__module__
+            parent, experiment_module = location.rsplit('.', 1)
+            module = import_module(parent + '.jupyter')
+        except (ImportError, ValueError):
```

Comment:
FWIW in the port-to-python-3 branch I've changed things so that `dallinger_experiment` gets imported as a package, so we should be able to use relative imports once that is merged.

Reply:
That'd be nice!